browser/model/installable-item.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
+let path = require('path');
+let reqs = require(path.resolve('./requirements.json'));
```

Comment:
Here is snippet that works https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/main.js#L79